### DISCUSSION TITLE
Clear mouse selection

### DISCUSSION THREAD

**Discussion by tamasfe** [zellij-org/zellij#2075] [Q&A] [answered]
After clicking anywhere on the terminal a new selection is started each time that will expand as new text appears and the scrollback expands by itself even if I don't scroll.

I've not found a way to clear mouse selections, as clicking anywhere just starts a new selection that will persist indefinitely. I've disabled mouse features altogether but it would be nice to have. Is this a bug or am I missing something?

**Comment by tlinford** [comment, marked as answer]
Do you by chance have 'copy_on_select false` set?

**Reply by tamasfe** [reply]
Yes, I do.

**Reply by tlinford** [reply]
I'll check it out, last time I played with it I noticed something wasn't right but it passed my mind!

**Reply by tlinford** [reply]
fix on its way: https://github.com/zellij-org/zellij/pull/2086

**Reply by tamasfe** [reply]
Awesome, thank you!
